packages/dappmanager/src/modules/upnpc/getExternalIp.ts

```diff
@@ -1,21 +1,18 @@
 import isIp from "is-ip";
 import upnpcCommand from "./upnpcCommand";
-import { UpnpError } from "./upnpError";
+import { parseUpnpErrors } from "./upnpError";
 
 export default async function getExternalIp(): Promise<string> {
-  const getExternalIpCommand = "-l";
   try {
-    const res = await upnpcCommand(getExternalIpCommand);
+    const res = await upnpcCommand("-l");
     const externalIp = ((res || "").match(
       /ExternalIPAddress.=.((\d+\.?){4})/
     ) || [])[1];
     if (isIp(externalIp)) return externalIp;
     else throw Error("Wrong IP");
   } catch (e) {
+    const upnpError = parseUpnpErrors(e.message);
```

Comment:
The function signature is `parseUpnpErrors(terminalOutput: string)` however you are passing here the error message, which *should* contain the terminal output but it's prefixed with more things.